charts/reports/values.yaml

```diff
@@ -0,0 +1,55 @@
+# Default values for plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  tag: alpine
+  pullPolicy: IfNotPresent
```

Comment:
As you are using the tag 'alpine', which is not a "fixed" image, can you set the pullPolicy to Always.
We don't take a lot of risks to always pull the latest available image, the nginx configuration remains quite standard and even if something is wrong, reports.jenkins.io is not critical and monitored